### ISSUE TITLE
`FCCtrlCheckbox` - `ThreeStatesMode`

### ISSUE THREAD

**Issue by ThistleSifter** [finale-lua/lua-scripts#522]
I was testing this out and it seems to do nothing for me on Windows. Is it supposed to? Under what circumstances does it work? I'm working on getting control state preservation to work for checkboxes but I can't figure out how to account for `ThreeStatesMode`.

As an example, the following script shows `1` when I click the button:
```lua
local dialog = finale.FCCustomLuaWindow()
local check = dialog:CreateCheckbox(10, 10)
check:SetThreeStatesMode(true)
check:SetCheck(2)
local btn = dialog:CreateButton(10, 40)
dialog:RegisterHandleControlEvent(btn, function()
    finenv.UI():AlertInfo(tostring(check:GetCheck()), "Check")
end)
dialog:ExecuteModal(nil)
```
@rpatters1

**Comment by rpatters1** [comment]
It will be implemented for Windows in the next release of RGP Lua.

**Comment by ThistleSifter** [comment]
Alrighty, thanks.

**Comment by jwink75** [comment]
I haven't actually tested it, but at least according to the release notes Robert addressed this in RGP Lua 0.68. I will close this thread unless there are any further objections!
